chapter10/autonomous-phone-registration/run_acceptance.py

```
@@ -0,0 +1,239 @@
+    runtime_files = [
+        "browser.py", "bus.py", "decision.py", "demo.py", "models.py",
+        "orchestration.py", "run_acceptance.py", "voice.py", "webrtc_channel.py",
+    ]
+    artifacts = [report_path, decision_path, timeline_path, receipt_path]
+    manifest = {
+        "schema_version": 1,
+        "experiment": "10-5",
+        "run_kind": "full_safe_webrtc_acceptance",
+        "generated_at": time.strftime("%Y-%m-%dT%H:%M:%S%z"),
+        "git_head_at_run": subprocess.run(
+            ["git", "rev-parse", "HEAD"], cwd=root, check=True, capture_output=True, text=True
```

Ruff (0.16.0)

[error] Starting a process with a partial executable path

(S607)